.github/workflows/ci.yml

```
@@ -0,0 +1,22 @@
+name: Continuous integration
+
+on:
+  push:
+    branches-ignore:
+      - "master"
+      - "develop"
+      - "release/**"
+  pull_request:
+
+jobs:
+  test:
+    name: test
+    runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-latest
```
There is not need to use 20.04. In addition, 20.04 will become latest soon.